### ISSUE TITLE
Support albums split into multiple directories (for multi-disc albums)

### ISSUE THREAD

**Issue by afontenot** [sentriz/gonic#374]
gonic version: d6b35f8c79bfa3c717582573ec69bfe8079ffb36

The README says:

> when browsing by folder, any arbitrary and nested folder layout is supported, with the following caveats:
> * Files from the same album must all be in the same folder
> * All files in a folder must be from the same album

This is slightly misleading. The limitations on directory structure do not just affect folder browsing, they (appear to) affect everything in gonic. As a result I've got a bunch of duplicated albums even when browsing by tags.

I'd like to see this restriction loosened ever so slightly. I don't need support for the "bucket of unsorted files" approach seen in https://github.com/sentriz/gonic/issues/89, rather it would be sufficient to change the requirements above to "all files from an album must be in a folder, or a subfolder of that folder, which does not contain files from any other album".

In other words, something like this:

```
Artist/Album
├── Disc 01
│   ├── 01 - Track Name.flac
│   ├── 02 - Track Name.flac
└── Disc 02
    ├── 01 - Track Name.flac
    ├── 02 - Track Name.flac
```

It's worth noting that beets (the tagging software) [specifically special cases](https://beets.readthedocs.io/en/stable/guides/tagger.html) for multi-disc usage, and that would work for me equally well:

> directories that look like separate parts of a *multi-disc album* are tagged together as a single release. If two adjacent albums have a common prefix, followed by “disc,” “disk,” or “CD” and then a number, they are tagged together.

**Comment by sentriz** [comment]
hey. yes as outlined in the readme, one album needs to be one folder. this is the tradeoff gonic needed to make to make browsing by tags and browsing by folder exist at the same time. it's far too much work to go back on this and support both modes at the same time. i personally don't have any interest in working on that. sorry!

for multidisc stuff i just encode the disc in the filenames eg

album/d01 tr01 a.flac
album/d01 tr02 b.flac
album/d01 tr03 c.flac
album/d02 tr01 a.flac
album/d02 tr02 b.flac
album/d02 tr03 c.flac
